spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/reload/ConfigurationUpdateStrategy.java

```diff
@@ -26,6 +26,13 @@
  */
 public record ConfigurationUpdateStrategy(String name, Runnable reloadProcedure) {
 
+	/**
+	 * Strategy that does nothing.
+	 */
+	public static final ConfigurationUpdateStrategy NOOP = new ConfigurationUpdateStrategy("no-op", () -> {
```

Comment:
this is needed for configuration watcher, when reload is disabled. In that case, we do not care about a `ConfigurationUpdateStrategy`. The only reason we need it, is because we extend various classes that need it via the calls to `super`, but in the configuration watcher itself, we make no use of it. 

I can go into the details here, if needed, just let me know.